lib/urlwatch/storage.py

```diff
@@ -77,6 +77,11 @@
                 'keyring': True,
             },
         },
+        'pushover': {
+            'enabled': False,
+            'app': '',
+            'user': ''
+            }
```

Comment:
Please fix the indentation of the closing `}` and add a `,` after it so that adding new sections makes the diff nicer.